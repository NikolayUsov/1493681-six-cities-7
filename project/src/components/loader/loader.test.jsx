import React from 'react';
import { render } from '@testing-library/react';
import Loader from './Loader';


describe('Test Map component', () => {

  it('Should render correctly', () => {
    const { asFragment } = render(<Loader />);
    expect(asFragment()).toMatchSnapshot();
  });
});
