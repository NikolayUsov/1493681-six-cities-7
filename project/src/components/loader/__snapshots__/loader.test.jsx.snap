// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Map component Should render correctly 1`] = `
<DocumentFragment>
  <span
    class="css-1dcri9m"
  >
    <span
      class="css-vdd3er"
    />
    <span
      class="css-119ohoe"
    />
  </span>
</DocumentFragment>
`;
